### DISCUSSION TITLE
Installazione HA core su docker e raspberry 4 pi

### DISCUSSION THREAD

**Discussion by jovannig** [indomus/forum#778] [HUB: Home Assistant]
Ho seguito passo passo i comandi per l'installazione, alla pagina:
https://indomus.it/guide/come-installare-e-configurare-home-assistant-core-con-docker-su-un-raspberry-pi-gia-in-uso/#docker

Pero' quando do' il comando per istanziare il container (docker-compose up -d home-assistant) il processo dura ore e ore senza arrivare alla fine
se digiti docker ps ottengo questo;
CONTAINER ID   IMAGE                                          COMMAND   CREATED         STATUS                          PORTS     NAMES
6405222e226d   ghcr.io/home-assistant/home-assistant:stable   "/init"   6 minutes ago   Restarting (0) 36 seconds ago             home-assistant

se leggo i log di installazione col comando docker container logs home-assistant -f, mi da' il seguente resoconto:

current thread 0xb6f9b020 (most recent call first):
<no Python frame>
[cont-finish.d] executing container finish scripts...
[cont-finish.d] done.
[s6-finish] waiting for services.
[s6-finish] sending all processes the TERM signal.
[s6-finish] sending all processes the KILL signal and exiting.
[s6-init] making user provided files available at /var/run/s6/etc...exited 0.
[s6-init] ensuring user provided files have correct perms...exited 0.
[fix-attrs.d] applying ownership & permissions fixes...
[fix-attrs.d] done.
[cont-init.d] executing container initialization scripts...
[cont-init.d] done.
[services.d] starting services
[services.d] done.
Fatal Python error: init_interp_main: can't initialize time
Python runtime state: core initialized
PermissionError: [Errno 1] Operation not permitted

Current thread 0xb6fbc020 (most recent call first):
<no Python frame>
[cont-finish.d] executing container finish scripts...
[cont-finish.d] done.
[s6-finish] waiting for services.
[s6-finish] sending all processes the TERM signal.
[s6-finish] sending all processes the KILL signal and exiting.

**Comment by snakuzzo** [comment]
Leggi [QUI](https://indomus.it/news/aggiornamento-2021-7-per-home-assistant-core/).
Devi aggiornare la libreria `libseccomp` sull'host...l'abbiamo scritto nella nota di aggiornamento della release di luglio...

![capture_2021-08-28_01-52](https://user-images.githubusercontent.com/7205371/135000526-db3738cd-8c33-44be-b7a6-bfa9188bd8b7.png)

ed era indicato anche nella nota ufficiale quando si è passati alle immagini Docker basate su Alpine 3.13

![aaaa](https://user-images.githubusercontent.com/7205371/135000723-0f0b862c-893c-4de9-8332-960c8245ffaa.png)

Puoi farlo direttamente dal package manager, ma devi vedere se ti installa una versione superiore alla 2.42.

```
sudo apt-get upgrade libseccomp2
```

Se invece la versione è precedente puoi installarlo a mano scaricando ed installando una versione più recente...

Es.

```
wget https://ftp.debian.org/debian/pool/main/libs/libseccomp/libseccomp2_2.5.1-1_armhf.deb
sudo dpkg -i libseccomp2_2.5.1-1_armhf.deb
rm libseccomp2_2.5.1-1_armhf.deb
```
